app/controllers/concerns/curation_concerns/admin_controller_behavior.rb

```diff
@@ -0,0 +1,33 @@
+module CurationConcerns
+  module AdminControllerBehavior
+    extend ActiveSupport::Concern
+
+    included do
+      cattr_accessor :configuration
+      self.configuration = CurationConcerns.config.dashboard_configuration
+      before_action :require_permissions
+      before_action :load_configuration
+      layout "admin"
+
+      def index
+        render "index"
```

Comment:
Yeah, but we also need the line below. Will `render "index"` still fire automatically?